### PR TITLE
VDS-1643: Rename vc-module-simple-export-import to vc-module-price-export-import

### PR DESCRIPTION
**Pull request by pushnitsa**
Description: 

--

QA-test: VDS-1644

Demo-test: VDS-1645

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.PriceExportImport_1.1.0-pr-50-a9e6694c.zip

### CODE REVIEW

File: src/VirtoCommerce.PriceExportImportModule.Web/Content/css/simple-export-import.css [hunk 1]
```diff
@@ -0,0 +1,176 @@
+.price-export-import .d-flex {
```
Comment (trueboroda) [inline src/VirtoCommerce.PriceExportImportModule.Web/Content/css/simple-export-import.css]: You can rename the file.

File: src/VirtoCommerce.PriceExportImportModule.Web/Scripts/dialogs/priceExport-dialog.tpl.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<div class="modal-header price-export-import" ng-class="{'modal-warning': validationError}">
```
Comment (trueboroda) [inline src/VirtoCommerce.PriceExportImportModule.Web/Scripts/dialogs/priceExport-dialog.tpl.html]: you can use the kebab style naming for file name

**Review by trueboroda — APPROVED** [review]

**Review by Pav-el — APPROVED** [review]

**Review by asvishnyakov — CHANGES_REQUESTED** [review]
Not renamed repo in mkdocs.yml
Not renamed solution in deploy-pr.yml and main.yml
Not renamed environment variables in index.md

**Review by asvishnyakov — CHANGES_REQUESTED** [review]
https://github.com/VirtoCommerce/vc-module-price-export-import/pull/50/files#diff-0d2b8fc76ba55e5a5b4505cfcaa6e05ce79b7d0a54f657581e23315419a3e1fbR60
Change this name too, please

File: src/VirtoCommerce.PriceExportImportModule.Core/ModuleConstants.cs [hunk 3]
```diff
@@ -59,26 +59,26 @@ public static class General
             {
                 public static SettingDescriptor ExportLimitOfLines { get; } = new SettingDescriptor
                 {
-                    Name = "SimpleExportImport.Export.LimitOfLines",
-                    GroupName = "SimpleExportImport|Export",
+                    Name = "PriceExportImport.Export.LimitOfLines",
```
Comment (asvishnyakov) [inline src/VirtoCommerce.PriceExportImportModule.Core/ModuleConstants.cs:62]: Don't forget to change https://github.com/VirtoCommerce/vc-webstore-deploy/blob/qa/webstore-app/resources/platform-cm.yaml#L29-L31 or add info to QA issue

**Review by asvishnyakov — CHANGES_REQUESTED** [review]
Add info about incompatible VirtoCommerce.SimleExportImport module version into the manifest, to prevent installation of both old and renamed module

File: src/VirtoCommerce.PriceExportImportModule.Core/ModuleConstants.cs (hunk 3 above)
Comment (pushnitsa) [reply src/VirtoCommerce.PriceExportImportModule.Core/ModuleConstants.cs:62]: Done https://virtocommerce.atlassian.net/browse/VDS-1701

**Review by asvishnyakov — APPROVED** [review]

**Comment by asvishnyakov** [comment]
Update version to 1.0.1/1.1.0 please

**Pull request merged**